src/olympia/search/tests/test_search_ranking.py

```diff
@@ -725,7 +725,7 @@ def test_scenario_open_image_new_tab(self):
             'Open Image in New Tab',
             (
                 ['Open Image in New Tab', 5553],
-                ['Open image in a new tab', 1735],
+                ['Open image in a new tab', 1736],
```

Comment:
Why did this PR require these changes?

Reply:
it's the magic of ElasticSearch... the match score is calculated based on all the text content and that includes fields that contain an external url, so have outgoing server url applied to them.  As the outgoing server has changed the text is different so the score can be different.  A small numeric difference (that doesn't change the search ordering) is fine.